pulp_ansible/app/urls.py

```diff
@@ -249,5 +249,12 @@
     path(GALAXY_API_ROOT + "v1/", include(v1_urls)),
     path(GALAXY_API_ROOT + "v3/", include(v3_urls)),
     path(GALAXY_API_ROOT, GalaxyVersionView.as_view()),
-    re_path(r"^pulp/api/v3/ansible/copy/$", CopyViewSet.as_view({"post": "create"})),
+    path(
+        settings.V3_API_ROOT_NO_FRONT_SLASH + "pulp_ansible/tags/",
+        TagViewSet.as_view({"get": "list"}),
+    ),
+    path(
+        settings.V3_API_ROOT_NO_FRONT_SLASH + "ansible/copy/",
```

Comment:
Good change on the hardcoded API_ROOT! Shame we used 'ansible/' for copy and 'pulp_ansible/' for tags.